activerecord/lib/rails/generators/active_record/model/model_generator.rb

```diff
@@ -5,7 +5,7 @@
 module ActiveRecord
   module Generators # :nodoc:
     class ModelGenerator < Base # :nodoc:
-      argument :attributes, type: :array, default: [], banner: "field[:type][:index] field[:type][:index]"
+      argument :attributes, type: :array, default: [], banner: "field[:type][:required][:index] field[:type][:required][:index]"
```

Comment:
I generally don't think it's worth it to add more complex mapping to the generators. @dhh do you have any opinions about this extension?

And required is now a third naming for the model's presence and the databases `null: false`.

Reply:
I don't really like this for the CLI, but I do like the path it's shining for a conductor-driven UI that lets you use the generators.

Reply:
Cool, let's save it for the conductor then. @yhirano55 if you want to take a stab at making generators available for https://github.com/rails/rails/pull/35489, you've got the go ahead 😊